### PR TITLE
zh-translation: content/zh/docs/setup/install/multicluster/multi-primary_multi-network

### PR DESCRIPTION
**Pull request by nicole-lihui**
[ ] Configuration Infrastructure
[X] Docs
[ ] Installation
[ ] Networking
[ ] Performance and Scalability
[ ] Policies and Telemetry
[ ] Security
[ ] Test and Release
[ ] User Experience
[ ] Developer Infrastructure

### CODE REVIEW

**Review by SataQiu — COMMENTED** [review]
/ok-to-test

File: content/zh/boilerplates/multi-cluster-with-metallb.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+---
+{{< tip >}}
+如果你在 `kind` 中测试多集群设置，可以使用 [MetalLB](https://metallb.universe.tf/installation/) 让 `LoadBalancer` 服务使用 `EXTERNAL-IP`。
```
Comment (kebe7jun) [inline content/zh/boilerplates/multi-cluster-with-metallb.md:4]:
```suggestion
如果你在 `kind` 中测试多集群，可以使用 [MetalLB](https://metallb.universe.tf/installation/) 让 `LoadBalancer` 服务使用 `EXTERNAL-IP`。
```

File: content/zh/docs/setup/install/multicluster/multi-primary_multi-network/index.md [hunk 2]
```diff
@@ -152,7 +156,7 @@ $ kubectl --context="${CTX_CLUSTER2}" apply -n istio-system -f \
 
 ## 启用端点发现 {#enable-endpoint-discovery}
 
-在 `cluster2` 中安装一个提供 `cluster1` 访问权限的远程 secret。
+在 `cluster2` 中安装一个提供 `cluster1` API Server 访问权限的远程 secret。
```
Comment (kebe7jun) [inline content/zh/docs/setup/install/multicluster/multi-primary_multi-network/index.md:159]: Secret

File: content/zh/docs/setup/install/multicluster/multi-primary_multi-network/index.md [hunk 3]
```diff
@@ -161,7 +165,7 @@ $ istioctl x create-remote-secret \
   kubectl apply -f - --context="${CTX_CLUSTER2}"
 {{< /text >}}
 
-在 `cluster1` 中安装一个提供 `cluster2` 访问权限的远程 secret。
+在 `cluster1` 中安装一个提供 `cluster2` API Server 访问权限的远程 secret。
```
Comment (kebe7jun) [inline content/zh/docs/setup/install/multicluster/multi-primary_multi-network/index.md:168]: Secret

**Review by kebe7jun — APPROVED** [review]
lgtm

**Pull request merged**